app/src/store/cart.ts

```diff
@@ -300,13 +300,19 @@ export default function useCartStore() {
     return summary.slice(0, -3); // trim the trailing ` + ` from the string
   });
 
+  /**
+   * @notice Returns the amount of items in the cart
+   */
+  const cartItemsCount = computed(() => Object.entries(cartSummary.value).length);
```

Comment:
`cartSummary` condenses e.g. two cart items of 2 DAI and 3 DAI into a summary of "5 DAI", so using this for the count will return the wrong information. `cart` stores the full cart data, so this should be changed as follows.
```suggestion
  const cartItemsCount = computed(() => cart.value.length);
```

And if you want, for brevity, this getter is simple enough that you could even inline it like this:

```typescript
return {
  cart,
  cartItemsCount: computed(() => cart.value.length),
  // other stuff in the return block
}
```

Reply:
Oh, I see it now. Thanks. Fixed.